packages/card/style.module.css

```diff
@@ -4,20 +4,13 @@
   --secondary-text-color: var(--wpl-neutral-500);
   --background-color: var(--wpl-neutral-0);
 
-  border: 1px solid var(--border-color);
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
```

Comment:
I think this is still needed at the root level to ensure the card grows the the height of containing element. See the last card is not the same height.

<img width="1485" alt="CleanShot 2022-10-26 at 13 23 55@2x" src="https://user-images.githubusercontent.com/825855/198094576-c8e99df3-2558-4dbd-a05a-23c79eedfa58.png">

Reply:
❗ Yep, absolutely needs to stay there as well :+1: 

Addressed here:
https://github.com/hashicorp/react-components/pull/776/commits/dd5925b910769a3050a322764b0973e7af208aec